### DISCUSSION TITLE
A fool question, but I cant find where to define page rules.

### DISCUSSION THREAD

**Discussion by Seashor** [requarks/wiki#2934] [Help / Questions] [answered]
Happy to join the wiki.js community. 
Wiki.js is awesome. But I have to restrict users from access other groups' page.
I learned that page rules can help. But I cannot find any entry to define page rules  as administator user .
Please someone helps me with that.

**Comment by NGPixel** [comment, marked as answer]
Under Administration > Groups

**Reply by Seashor** [reply]
Thanks bro !
